src/git.rs

```diff
@@ -316,6 +414,36 @@ pub fn branch() -> String {
 /// When current repository exists git folder.
 /// I's use [Command] to get.
 pub fn tag() -> String {
+    #[cfg(feature = "gitoxide")]
```

Comment:
I couldn't find where the original `command_current_tag()` is defined but I think this implementation would make me *not* want to use `gix` 😅. 

Let's mark this as 'needs work' and potentially figure out other ways to write the same or see if this is a call for a new method in the `gix` API.